docs/guides/monitoring/notifications.md

```diff
@@ -22,8 +22,8 @@ The Serverless Framework can notify you in **Slack**, **Email**, **SNS Topics**,
 3. Open the **notifications** tab in the application.
 4. Follow the "**Click here to create your first notification.**" link.
 5. Fill out the form and click **add notification** to save the new notification. The notification can be scoped to include only the **alerts**, **stages** or **services** you want.
-   - **alerts** - select "all alerts" to be notified about all alerts, including alerts which may be made available in the future, or individually select the alerts. The [Alerts](./insights.md#alerts) section provides more details on each of the available alert types.
-   - **stages** - select "all stages" to be notified about alerts on all stages, including stages which may be created in the future, or individually select the stages. The stages must be [configured](./profiles.md#add-a-deployment-profile-to-your-application-and-stage) on the application first for them to be available.
+   - **alerts** - select "all alerts" to be notified about all alerts, including alerts which may be made available in the future, or individually select the alerts. The [Alerts](../insights.md#alerts) section provides more details on each of the available alert types.
```

Comment:
In this case I think right url is `./alerts.md`

Reply:
Ohh yeah, huge mistake, couldnt spot really that🙏, will make the changes